ansible/roles/debops.nsswitch/templates/etc/ansible/facts.d/nsswitch.fact.j2

```diff
@@ -9,7 +9,7 @@ import os
 
 nsswitch_conf = '/etc/nsswitch.conf'
 
-nss_databases = loads('''{{ nsswitch__combined_database_map.keys() | to_nice_json }}''')
+nss_databases = loads('''{{ nsswitch__combined_database_map | list | to_nice_json }}''')
```

Comment:
Despite being correct, I feel `nsswitch__combined_database_map | list` takes out a little bit of readability.

How do you feel about using `nsswitch__combined_database_map.keys() | list`, even if it's not as efficient?

Reply:
Well, but that makes the lines even loger. I am not sure if that supports readability.

Reply:
It's a little bit unexpected to the casual eye that `list` on a map returns its keys, having `keys` is more explicit and reassuring. It's just nitpicking though, I don't have a strong opinion about it.